core/arch/arm/tee/entry_fast.c

```diff
@@ -103,6 +103,13 @@ static void tee_entry_exchange_capabilities(struct thread_smc_args *args)
 
 	args->a0 = OPTEE_SMC_RETURN_OK;
 	args->a1 = OPTEE_SMC_SEC_CAP_HAVE_RESERVED_SHM;
+
+	if (core_mmu_nsec_ddr_is_defined()) {
+		IMSG("NS DDR configured. Enabling dynamic SHM");
```

Comment:
minor: s/configured/defined/. This code does not really configure anything.
Same below.